packages/global/config/omeda-identity-x.js

```diff
@@ -60,4 +60,21 @@ module.exports = ({
   appendPromoCodeToHook,
   appendBehaviorToHook,
   appendDemographicToHook,
+  onLoginLinkSentFormatter: (async ({ req, payload }) => {
+    const omeda = req.app.locals.site.getAsObject('omeda');
+    const { source } = req.body;
+    const { onLoginLinkSentPromoCodes } = omeda;
+    const { default: defaultPromoCode } = onLoginLinkSentPromoCodes.default;
```

Comment:
This looks wrong -- is your variable structure `onLoginLinkSentPromoCodes.default.default`?